plugins/modules/gitlab_project.py

```diff
@@ -162,6 +162,14 @@
     type: bool
     default: false
     version_added: "4.0.0"
+  issues_access_level:
+    description:
+      - V(private) means that accessing issues tab is allowed only to project members.
+      - V(disabled) means that accessing issues tab is disabled.
+      - V(enabled) means that accessing issues tab is enabled.
```

Comment:
```suggestion
      - V(enabled) means that accessing issues tab is enabled.
      - O(issues_access_level) and O(issues_enabled) are mutually exclusive.
```
Please also add this to the `issues_enabled` documentation.